pkg/runtime/wfengine/workflow.go

```diff
@@ -391,10 +399,12 @@ func (wf *workflowActor) runWorkflow(ctx context.Context, actorID string, remind
 			return newRecoverableError(errExecutionAborted)
 		}
 	}
+	wfLogger.Debugf("workflow actor '%s': workflow execution returned with status %s instanceId '%s'", actorID, runtimeState.RuntimeStatus().String(), wi.InstanceID)
```

Comment:
single quotes around the %s after status

Reply:
Done